Project.toml

```
@@ -78,7 +78,7 @@ OffsetArrays = "1.4"
 OrderedCollections = "1.1"
 Printf = "1.9"
 Random = "1.9"
-Reactant = "0.2.31"
+Reactant = "0.2.38"
```

Maybe 
```suggestion
Reactant = "0.2.40"
```
? 0.2.39 is borked with Oceananigans